Sources/SwiftASN1/Basic ASN1 Types/GeneralizedTime.swift

```diff
@@ -105,6 +105,24 @@ public struct GeneralizedTime: DERImplicitlyTaggable, BERImplicitlyTaggable, Has
         }
         set {
             self._fractionalSeconds = newValue
+            self._rawFractionalSeconds = ArraySlice<UInt8>()
+            try! self._rawFractionalSeconds.append(fractionalSeconds: self._fractionalSeconds)
+
+            try! self._validate()
+        }
+    }
+
+    /// The ArraySlice of bytes from which the fractional seconds will be computed. (Preserved due to a possible overflow
+    /// when computing a Double from this ArraySlice.)
+    @inlinable
+    public var rawFractionalSeconds: ArraySlice<UInt8> {
```

Comment:
I'm worried about crashing in the setter. Throwing is okay but crashing isn't great, especially without an easy non-throwing version.

@Lukasa how do you feel about deprecating the setters and replacing them with throwing set methods?
We already had the force try in the previous implementation. We should at least document the precondition.

Reply:
Do we even have a  public (throwing) initializer that takes `_rawFractionalSeconds`?
I don't see it in the diff.

Reply:
> Do we even have a public (throwing) initializer that takes _rawFractionalSeconds?

We don't. To be fair, I told Clinton I didn't think we needed one. But you're right that we should be consistent.

As for crashing in the setter, sometimes that's just a requirement of this flow.